Cli/Basic.lean

```diff
@@ -1,5 +1,9 @@
+module
+
 import Std.Data.TreeSet
```

Comment:
I think if we want to be overly careful not to break anything, this should still be `public`. Alternatively, we should also remove the `public` on the `Std` imports in `Cli.Extensions`.